aiinfra-cluster/variables.tf

```diff
@@ -90,23 +90,65 @@ variable "disk_size_gb" {
 }
 
 variable "disk_type" {
-  description = ""
+  description = "Boot disk type, can be either pd-ssd, local-ssd, or pd-standard."
   type        = string
+  default     = "pd-standard"
+
+  validation {
+    condition     = contains(["pd-ssd", "local-ssd", "pd-standard"], var.disk_type)
+    error_message = "Variable disk_type must be one of pd-ssd, local-ssd, or pd-standard."
+  }
 }
 
 variable "network_config" {
-  description = ""
+  description = "The network configuration to specify the type of VPC to be used"
```

Comment:
perhaps enumerate the valid options (as above) and specify which one is the default